### PR TITLE
add a module to spend cpu time

### PR DESCRIPTION
**Pull request by davidlange6**
to reduce I/O / per second if desired

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: CommonTools/Utils/plugins/CPUSpender.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#ifndef CommonTools_Utils_CPUSpender_h
+#define CommonTools_Utils_CPUSpender_h
+
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+class CPUSpender: public edm::EDAnalyzer {
```
Comment (slava77) [inline CommonTools/Utils/plugins/CPUSpender.h]: is it a legacy module on purpose?

**Comment by davidlange6** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/14893 	bc06946

80X version is merged already, but auto-fwd-port is stuck https://cmssdt.cern.ch/jenkins/job/auto-forward-ports/1746/console. Otherwise, this PR would already turn to no-op.

**Pull request merged**